sway/desktop/render.c

```diff
@@ -884,7 +884,11 @@ static void render_view(struct sway_output *output, pixman_region32_t *damage,
 		scale_box(&box, output_scale);
 		int scaled_corner_radius = deco_data.corner_radius == 0 ?
 				0 : (deco_data.corner_radius + state->border_thickness) * output_scale;
-		float* shadow_color = view_is_urgent(view) || state->focused ?
+		bool has_inactive_color = false;
+		for (int i = 0; i < 4; i++) {
+			has_inactive_color |= config->shadow_inactive_color[i] != 0;
+		}
```

Comment:
Keeping it in the command file is a better solution. Then this doesn't have to run every time we render something

Reply:
Sorry, that makes a lot of sense. Should be fixed now

Reply:
> Sorry, that makes a lot of sense. Should be fixed now

Doesn't seem to work. The first `memcmp` implementation worked best. But as in that comment, check if the result equals 0 or not

Reply:
Ok, I'm so sorry. I completely misinterpreted what you meant by inverting the value and thought I had to do it another way. I feel a bit silly now ahaha. It's back to the first one but using equals zero and with brackets. Sorry for all the fuss!